cornell-dti/course-plan
# Add ansc minor

### Summary 






This pull request is to add the Animal Science Minor. This minor previously existed (https://cals.cornell.edu/animal-science/degrees-programs/undergraduate-studies/minor-requirements), however was not yet implemented.
- [x] implemented Animal Science minor

### Test Plan 
- Test if ANSC 4270 counts towards the animal science minor
- Confirm whether BIOAP 1100 counts towards the minor
- Confirm MATH 1910 does not count towards Animal Science Minor



### Notes 

- Anytime a new Animal Science course is added for the Animal Science minor, "src/data/minors/animal-sci.ts" should update to contain it.
- Minor offered by AG (CALs).
